REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.6
-Compat 0.30.0
+Compat 0.38.0
```

Comment:
Compat 0.39

Reply:
It doesn't depend on anything in Compat 0.39 that's not in Compat 0.38, so I think this should be left.

Reply:
Compat 0.38 is broken. Please update to 0.39.

Reply:
I wrote this and then I merged 20 seconds later. Oh well, will update on master.